src/styles/blocks/cards-container.scss

```diff
@@ -0,0 +1,10 @@
+.cards-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 0 auto;
+  padding: 50px 40px 73px;
+  @media (min-width: 1250px) {
```

Comment:
I'd like to add an empty line before the directives and mixins (if they are not the first rule in the block)